cv32e40s/sim/tools/xrun/cv32e40s.manual_expression_cov_todo.vRefine

```diff
@@ -1,164 +1,197 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <refinement-file-root>
-  <information comment-version="1" creation-time="Fri 08 Sep 2023 16:33:54 CEST" creator="krdosvik" csCheck="true" modified-by="krdosvik" modify-time="Fri 22 Sep 2023 15:00:52 CEST" save-ref-method="seq" tool-version="Cadence vManager22.05" rules-signature-c="5ec06b80ed3aa56a354242dc7124e080">
```

Comment:
Hem, that I have changed this file was a bit unexpected.

Reply:
manual_expression_cov_todo.vRefine seems to have changes that was also stored and modified (for the better) in the xsecure_trigger file.